Build/NuGet/.pack-version

```diff
@@ -0,0 +1 @@
+2.0.2-pre
```

Comment:
@digitalinfinity what do you think? IMO it's not necessary to change this for this PR. We will update it as needed.